src/Spryker/Zed/ProductPackagingUnitStorage/Persistence/ProductPackagingUnitStorageRepository.php

```diff
@@ -90,6 +90,19 @@ public function findFilteredProductPackagingUnit(FilterTransfer $filterTransfer,
         return $this->buildQueryFromCriteria($query, $filterTransfer)->find();
     }
 
+    /**
+     * @param \Generated\Shared\Transfer\FilterTransfer $filterTransfer
+     *
+     * @return \Generated\Shared\Transfer\ProductPackagingLeadProductTransfer[]
+     */
+    public function getProductPackagingLeadProductTransfersByFilter(FilterTransfer $filterTransfer): array
```

Comment:
It will not return what you expect, instead of `ProductPackagingLeadProductTransfer[]` will be `SpyProductPackagingLeadProductEntityTransfer[]`.